tldw_Server_API/app/api/v1/endpoints/audio.py

```
@@ -1398,17 +1454,60 @@ def _on_audio(seconds: float, sr: int) -> None:
+                    _QuotaExceeded: If adding this chunk would exceed the user's daily minutes quota.
+                
+                Notes:
+                    - Checks whether the user's remaining daily minutes allow this chunk; if allowed, increments the nonlocal
+                      `used_minutes` counter and records the minutes via `add_daily_minutes`.
+                """
                 nonlocal used_minutes
                 minutes_chunk = float(seconds) / 60.0
-                allow, _ = await check_daily_minutes_allow(user_id_for_usage, minutes_chunk)
+                try:
+                    allow, _ = await check_daily_minutes_allow(user_id_for_usage, minutes_chunk)
+                except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)